### DISCUSSION TITLE
关于CM311-1Sa S905L3-A 适配问题

### DISCUSSION THREAD

**Discussion by Robert201212** [ophub/amlogic-s9xxx-armbian#1595] [Amlogic]
O大，你好！
  我目前下载了点心云斐讯N1的固件，这个固件是基于6.1.32斐讯版本改过的。目前我想把它刷入CM311-1SA里面，更换DTB以及bin文件，可以进行正常安装，但是启动后就会卡在界面，无法正常工作。经查看还是在调用meson-gxl-s905d-phicomm-n1.dtb、u-boot-n1.bin、u-boot-2015-phicomm-n1.bin，所导致的无法开机。

目前如果直接给CM311-1sa刷入最新版本，EMMC以及U盘启动，均为正常。

给您发邮件了，盼复！

**Comment by ophub** [comment]
你是cm311为什么要使用N1的固件呢？
装CM311自己的

**Comment by Robert201212** [comment]
O大，我现在把CM311-1sa改装到128Gb了，用于跑CDN业务，业务方的镜像里面有他们的业务系统。所以要使用这个固件。请求o大给配置下，万分感谢！您给小弟个收款码，小弟200红包送上，微薄心意，还望收下。


业务方镜像(6.1.32)：https://fae-cdn.linkfog.cn/ios/armbian/n1/armbia-N1.7z


---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2023年08月07日 15:34 |
| 收件人 | ophub/amlogic-s9xxx-armbian ***@***.***> |
| 抄送人 | Robert201212 ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [ophub/amlogic-s9xxx-armbian] 关于CM311-1Sa S905L3-A 适配问题 (Discussion #1595) |

你是cm311为什么要使用N1的固件呢？

**Comment by ophub** [comment]
各固件的区别就是
/boot/uEnv.txt
/boot/u-boot.ext
不同固件用的不一样，参考model里的配置，你把img写入u盘，插电脑打开boot自己修改一下就可以了，uEnv改dtb名字，u-boot复制对应的文件的

**Comment by Robert201212** [comment]
感谢O大，方便留您个微信吗？
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2023年08月07日 17:02 |
| 收件人 | ophub/amlogic-s9xxx-armbian ***@***.***> |
| 抄送人 | Robert201212 ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [ophub/amlogic-s9xxx-armbian] 关于CM311-1Sa S905L3-A 适配问题 (Discussion #1595) |

各固件的区别就是
/boot/uEnv.txt
/boot/u-boot.ext
不同固件用的不一样，参考model里的配置，你电脑打开boot自己修改一下就可以了

**Comment by Robert201212** [comment]
O大，我按照您的意思改了，启动依然调用的为meson-gxl-s905d-phicomm-n1.dtb，及n1.bin。
安装正常，启动花屏，我要奔溃了！
 
附件1、2，安装中，启动后












---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2023年08月07日 17:02 |
| 收件人 | ophub/amlogic-s9xxx-armbian ***@***.***> |
| 抄送人 | Robert201212 ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [ophub/amlogic-s9xxx-armbian] 关于CM311-1Sa S905L3-A 适配问题 (Discussion #1595) |

各固件的区别就是
/boot/uEnv.txt
/boot/u-boot.ext
不同固件用的不一样，参考model里的配置，你电脑打开boot自己修改一下就可以了

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

从 网易邮箱大师 发来的云附件
|
| |
| |
|
| |
| 83FC48988C8F49C4927A6D42DE6ED11F.JPG |
| |
| 4.8M · 存在有效期 |
| |
| 下载 |
| |
| |
|
| |
| |
|
| |
| 17B1837C7AF643BFAC5621F71FBACD2B.JPG |
| |
| 4.6M · 存在有效期 |
| |
| 下载 |
| |
| |
